Automattic/jetpack
# Jetpack Setup Wizard: Adjust order of recommended features

Fixes #15969

#### Changes proposed in this Pull Request:

This PR changes the order of the recommended features. Previously they were sorted alphabetically, and now they are sorted according to the order of their position in `featureGroups`.

#### Is this a new feature or does it add/remove features to an existing part of Jetpack?
No

#### Does this pull request change what data or activity we track or use?


No

#### Testing instructions:





1. Add the following to your wp-config.php:
```
add_filter( 'jetpack_show_setup_wizard', '__return_true' );
add_filter( 'jetpack_pre_connection_prompt_helpers', '__return_true' );
```
2. Visit `/wp-admin/admin.php?page=jetpack#/setup/intro` and proceed through the flow, answering "business", choosing all income options, and saying "yes" to blogs. This will ensure every feature displays.
3. Verify that the order of the features under "Performance" matches the order given in issue #15969.

#### Proposed changelog entry for your changes:

None needed
